src/injected/visualization/svg-shape-factory.ts

```diff
@@ -97,11 +98,42 @@ export class SVGShapeFactory {
 
         if (tabOrder != null) {
             text.innerHTML = tabOrder.toString();
+        } else {
+            text.innerHTML = 'X';
```

Comment:
There is a possibility that the circle won't have any text (user toggles visualization off and then on again so the tab order disappears; errored circle would have no number in it).